uwblueprint/project-read-frontend
# Refactor family list to handle state in the home & sessions pages

### Notion ticket link



n/a



### Implementation description

- no UX changes
- refactors the FamilyList component to handle state in the Sessions/MainRegistration pages
- this is necessary because when we implement editing families/enrolments in the sidebar, we'll need to update the family in the table



### Steps to test

1. backend needs to include [this PR](https://github.com/uwblueprint/project-read-backend/pull/91)
1. open the registration & sessions pages and verify everything still behaves the same



### What should reviewers focus on?

- double checking things still work

### Checklist
